lib/Response/EmptyResponse.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Aerys\Response;
+
+use Aerys\Response;
+use Amp\ByteStream\InMemoryStream;
+
+class EmptyResponse extends Response {
+    public function __construct(array $headers = [], int $code = 200, string $reason = null) {
```

Comment:
Default status code for an empty response should probably be `204 No Content` instead of `200`

Reply:
Yep, oversight.